enum-resource/smuggler.yml

```diff
@@ -0,0 +1,36 @@
+---
+commands:
+  check: |
+    /opt/resource/wrapped/s3/check > response.json
+    # If it is the first run, just dispatch a - string to for 'in' to be triggered
+    jq 'if . == [] then [{ "version_id":"-"}] else . end' < response.json
```

Comment:
Yes,  I have checked that is the case. I hijacked the container in CI (`./bin/fly -t master hijack -c create-bosh-cloudfoundry/healthcheck-deployed sh`) and run

``` 
# /opt/resource/wrapped/s3/check <<EOF
{
  "source": {
    "bucket": "master-state",
    "region_name": "eu-west-1",
    "versioned_file": "healthcheck-deployed",
    "valid_values": ["yes", "no"],
    "default_values": ["no"]
  },
  "version": { "version_id":"4lO7GqPDXFewTWypE8cOkeZdcSxWYiW3" }
}
EOF 
[]
```

It does return empty list `[]`.

The same if you specify a missing file:

```

``` 
# /opt/resource/wrapped/s3/check <<EOF
{
  "source": {
    "bucket": "master-state",
    "region_name": "eu-west-1",
    "versioned_file": "something_random",
    "valid_values": ["yes", "no"],
    "default_values": ["no"]
  },
  "version": { "version_id":"4lO7GqPDXFewTWypE8cOkeZdcSxWYiW3" }
}
EOF 
[]
```